src/components/Modals/BuyItemSlotsModal/BuyItemSlotsModal.tsx

```diff
@@ -133,7 +116,7 @@ export default class BuyItemSlotsModal extends React.PureComponent<Props, State>
           <NetworkButton
             className={styles.acceptButton}
             primary
-            disabled={hasInsufficientMANA || isBuyingItemSlots || selectedTierId === undefined}
+            disabled={hasInsufficientMANA || isBuyingItemSlots || slotsToBuy === undefined}
```

Comment:
Good call, updated!
- Cancel button disabled while fetching price || buying slots
- I added `isFetchingSlotPrice` to the `disabled` attr for the Buy button but it was disabled if `slotsToBuy === undefined` which is the initial state and it can't be changed until the price is loaded so I think there is no visual change for that.